python/GangaLHCb/Lib/Applications/BenderBox.py

```diff
@@ -0,0 +1,520 @@
+    """
+    _schema = Schema(Version(1, 0), {
+        'scripts'   : FileItem   (
+        defvalue        = []      ,
+        sequence        = 1       ,
+        strict_sequence = 0       ,
+        doc             = """The names of ostap script files to be executed. The files are executed within ``ostap'' context. A copy will be made at submission time"""),
+        'commands'  : SimpleItem (
+        defvalue        = []      ,
+        typelist        = ['str'] ,
+        sequence        =  1      ,
+        doc             = """The ostap commands to be executed, e.g. [ 'print dir()' ]"""), 
+        'arguments' : SimpleItem (
+        defvalue        = []      ,
+        typelist        = ['str'] ,
+        sequence        =  1      ,
+        doc             = "The list of command-line arguments for ``ostap'' script, e.g. ['-w','-p5'], etc. Following arguments are appended automatically:  --no-color and --batch""")
+        })
+    _category      = 'BenderTool'
+    _name          = 'OstapRun'
+    _exportmethods = [ ]
+
+    layout = """#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# =============================================================================
+from distutils.spawn import find_executable
+ostap_script = find_executable('ostap')
+import sys
+# =============================================================================
+## redefine arguments 
+# =============================================================================
+sys.argv += {scripts}        
+sys.argv += [ '--no-color' ] ## automatically added argument
+sys.argv += {arguments}
+sys.argv += [ '--batch'    ] ## automatically added argument
+sys.argv += [ '--command'  ] + {command} 
+##
+# =============================================================================
+## execute bender script 
+# =============================================================================
+import runpy
+runpy.run_path ( ostap_script, init_globals = globals() , run_name = '__main__' )
+# =============================================================================
+# The END
+# =============================================================================
+"""
+    ## Returns file names which should be copyed to shared area
+    def inputfiles(self):
+        "Returns file names which should be copyed to shared area"
+        return [ LocalFile(f.name) for f in self.scripts ]
+
+    ## Returns  wrapper file which run at WN 
+    def getWrapper(self,data_file):
+        "Returns  wrapper file which run at WN "
+        
+        the_script    = self.layout.format (
+            scripts   = [ os.path.join ( f.subdir , os.path.basename ( f.name ) ) for f in self.scripts ] , 
+            arguments = self.arguments  ,
+            command   = self.commands    
+            )
+        return the_script
+
+## import the base class 
+from .GaudiExec import GaudiExec
+# =============================================================================
+## @class BenderBox
```

Comment:
In general I agree with @mesmith75 that these commented out lines add clutter, especially since it's mostly repeated in the docstrings anyway. A few lines for authorship credit (of course git will record your contribution anyway) are fine but I fail to see what the rest is adding?